mysql/cloudmysql/cloudmysql.go

```diff
@@ -79,6 +80,28 @@ func Open(ctx context.Context, certSource proxy.CertSource, params *Params) (*sq
 	return sql.OpenDB(connector(cfg.FormatDSN())), nil
 }
 
+// OpenGAE opens a Cloud SQL database on Google App Engine (GAE) using the
+// environment variables $DB_USER, $DB_DATABASE, $DB_INSTANCE, and
+// $DB_PASSWORD.
```

Comment:
Is this how GAE commonly does it, putting everything in environment variables?

Reply:
It's currently necessary because Go on GAE doesn't yet support an entrypoint field in app.yaml that would allow setting flags on the server.